var/spack/repos/builtin/packages/casper/package.py

```diff
@@ -22,6 +22,11 @@ class Casper(MakefilePackage):
 
     conflicts('%gcc@7.1.0')
 
+    def flag_handler(self, name, flags):
+        if name in ['cxxflags', 'CXXFLAGS']:
+            flags.append('-std=c++03')
+        return (flags, None, None)
```

Comment:
Can we use the the `cxx98_flag` property of the current compiler?

Reply:
Yeah, that's likely better.